HACKING.md

```diff
@@ -182,14 +182,33 @@ git submodules:
 Refer to the [testing README](test/README.md) for details on running the Cockpit
 integration tests locally.
 
-## Testing the Python bridge
+## Python bridge
 
 Cockpit currently has an experimental replacement for `cockpit-bridge`
 written in Python.  It resides in `src/cockpit` with most of its rules in
 `src/Makefile.am`.  This directory was chosen because it matches the standard
 so-called "src layout" convention for Python packages, where each package
 (`cockpit`) is a subdirectory of the `src` directory.
 
+### Running the bridge
+
+A test image with the Python image can be created with `image-prepare` for the operating systems which don't enable it yet.
+
+    ./test/image-prepare --python
+
+Or you can interactively run the bridge locally:
+
+    python3 -m cockpit.bridge
```

Comment:
This obviously needs `PYTHONPATH=src`

Reply:
Yep. @allisonkarlitskaya and I have two aliases:
```
alias cpf='PYTHONPATH=src python3 -m cockpit.misc.print'
alias cpy='PYTHONPATH=src python3 -m cockpit.bridge'
```